### PR TITLE
Fix notify reward amount

### PR DESCRIPTION
**Pull request by kinspotting**
Quick fix for `notifyRewardsAmount`, previously deployed to kovan `StakingRewards` contracts were not starting the distribution of tokens, this PR should fix it

### CODE REVIEW

File: contracts/incentives/FarmFactory.sol [hunk 1]
```diff
@@ -122,19 +119,6 @@ contract FarmFactory is CloneFactory {
 			address(avatar)
 		);
 
-		//call notify reward amount
-		(success,) = controller.genericCall(
```
Comment (BritikovKI) [inline contracts/incentives/FarmFactory.sol:126]: Why are we removing this call and require statement?

File: contracts/incentives/StakingRewards.sol [hunk 2]
```diff
@@ -73,11 +73,18 @@ contract StakingRewards is IRewardDistributionRecipient, ReentrancyGuard {
...
+        totalRewards = IERC20(_rewardToken).balanceOf(address(this));
+        rewardRate = totalRewards.div(duration);
+        lastUpdateTime = block.timestamp;
+        periodFinish = block.timestamp.add(duration);
+        emit RewardAdded(totalRewards);
     }
 
     uint256 public duration;
 
-    uint256 public initreward;
```
Comment (BritikovKI) [inline contracts/incentives/StakingRewards.sol:80]: Initreward is removed here and nowhere else, it wasn't used anywhere?

**Review by BritikovKI — CHANGES_REQUESTED** [review]

File: contracts/incentives/FarmFactory.sol (hunk 1 above)
Comment (kinspotting) [reply contracts/incentives/FarmFactory.sol:126]: I moved this functionality to `initialization`  function in `StakingRewards` contract

File: contracts/incentives/StakingRewards.sol (hunk 2 above)
Comment (kinspotting) [reply contracts/incentives/StakingRewards.sol:80]: I replaced it with `totalRewards`

**Comment by codebuster22** [comment]
@kinspotting The tests are passign, please review it once. Do not merge it yet. Once reviewed, I'll remove unnecessary comments and then we can merge it.

**Review by BritikovKI — APPROVED** [review]
Nice fix

**Pull request merged**